src/Bookings.js

```diff
@@ -1,19 +1,36 @@
-import React from "react";
+// ** Instructions:** In the`<Bookings />` component, declare a new state`bookings`
+//with the corresponding setter function `setBookings` to hold the`FakeBookings` data.
+//Instead of passing`FakeBookings` directly to the`<SearchResults />` component,
+//pass the new `bookings` state variable.
+
+// ** Hint:** The new `bookings` state should be initialised with the`FakeBookings`
+//variable.
+
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
+import SearchResults from "./SearchResults.jsx";
+
 // import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [BookingsData, setBokings] = useState([]);
```

Comment:
```
const [bookings, setBookings] = useState([]);
```

👍 well done with the useState hook! Be careful of little typos though, and the exercise was asking for a "bookings" variable. But it all works, so good stuff!